### ISSUE TITLE
News and interests still present.

### ISSUE THREAD

**Issue by GnomeBeans** [ChrisTitusTech/win10script#234]
I'm running as admin. Went to run essentials and news and interests still shows in the taskbar. This is after a reset of windows. I looked into your script and found the dword in the registry but it isn't changing.
EDIT: If I manually change it in the taskbar the dword updates but anytime I manually edit it in regedit it always seems to popup with a zero entered into the box.

**Comment by GnomeBeans** [comment]
This key doesn't exist for me
Set-ItemProperty -Path "HKCU:\SOFTWARE\Policies\Microsoft\Windows\Windows Feeds" -Name "EnableFeeds" -Type DWord -Value 0

When I did windows reset I skipped privacy settings. Maybe those are required to create the key idk.

**Comment by Articul8Madn3ss** [comment]
This is just another prime reason why Chris Titus needs to go back to the Chocolatey script.  This winget mess isn't working.

**Comment by GnomeBeans** [comment]
Small update. I've noticed some behavior that I cannot pinpoint exactly but I'm guessing it has to do with reset options cloud vs local. Recently ran the script again, essential option on my main machine after some windows updates because potential of o&o settings getting reverted. When I did this after rebooting windows took me through some setup menus like I was reinstalling windows. It seemed really odd. Nothing I couldn't skip but for some reason it makes me feel more confident that the script is working. I've not had to revert essentials option so I have no idea how effective that is and hope I don't need to. It seems unobtrusive enough that I can run it and leave it alone and many things that bug me about windows goes away. I trust this one the most and I've recommended to friends too.
